packages/mobile/patches/react-native-code-push+7.0.5.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/node_modules/react-native-code-push/android/codepush.gradle b/node_modules/react-native-code-push/android/codepush.gradle
+index 22be273..468fdb1 100644
+--- a/node_modules/react-native-code-push/android/codepush.gradle
++++ b/node_modules/react-native-code-push/android/codepush.gradle
+@@ -25,6 +25,17 @@ gradle.projectsEvaluated {
+     }
+     
+     android.applicationVariants.all { variant ->
++        // skip debug build variant

```

Comment:
Why is this patch needed?

Reply:
https://github.com/microsoft/react-native-code-push/issues/2418

Reply:
will add this as comment